Sample-WPF/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.ProjectOxford.Vision" version="1.0.372" targetFramework="net452" />
+  <package id="Microsoft.ProjectOxford.Vision" version="[need update]" targetFramework="net452" />
```

Comment:
This can be updated when the NuGet packages ship. Generally we do updates in two passes: one to make the client library, which we publish with a version number, then a second to update the sample app to use the new package.